src/users/users.service.ts

```diff
@@ -139,4 +148,37 @@ export class UsersService implements OnModuleInit {
     console.log({ userId });
     return this.userIdentityModel.findOne({ userId }).exec();
   }
+
+  async createUserJWT(
+    accessToken: JWTUser | undefined,
+  ): Promise<CreateUserJWT | null> {
+    const signAndVerifyOptions = {
+      expiresIn: this.configService.get<string>("jwt.expiresIn") || "1h",
+      secret: this.configService.get<string>("jwt.secret"),
+    };
+    const jwt = new JwtService(signAndVerifyOptions);
```

Comment:
JwtService should be added through the constructor, see [auth.service.ts](https://github.com/SciCatProject/scicat-backend-next/blob/master/src/auth/auth.service.ts).